src/json5-validation.ts

```diff
@@ -1,21 +1,22 @@
 import { EditorView } from "@codemirror/view";
-import { JSONValidation, type JSONValidationOptions } from "./json-validation";
-import type { JSONSchema7 } from "json-schema";
-import { parseJSON5DocumentState } from "./utils/parseJSON5Document";
+import { linter } from "@codemirror/lint";
+import {
+  JSONValidation,
+  handleRefresh,
+  type JSONValidationOptions,
```

Comment:
Did you have to manually update all the imports to use the .js extension?

Reply:
yes, this is part of adopting the new node module resolution spec!